.typedoc/custom-theme.mjs

```diff
@@ -359,8 +375,11 @@ class ClerkMarkdownThemeContext extends MarkdownThemeContext {
           // So, .someType adds the backticks to the string and we don't want to deal with modifying that partial. So just remove any backticks in the next step again :shrug:
           const defaultResult = this.partials.someType(unionType, { forceCollapse: true });
 
+          // Escape stuff that would be turned into markdown
+          const escapedResult = defaultResult.replace(/__experimental_/g, '\\_\\_experimental\\_');
```

Comment:
Previously this would have been wrapped in backticks, without it the markdown parser would turn this into `**experimental`